README.md

```diff
@@ -1,5 +1,7 @@
 # Cloud Foundry CLI Plugin Repository (CLIPR)[![Build Status](https://travis-ci.org/cloudfoundry/cli-plugin-repo.svg?branch=master)](https://travis-ci.org/cloudfoundry/cli-plugin-repo)
 
+[![](https://uptime.com/devices/services/widget/689899/a14fde9ba2e0bc1e/service?light)](https://uptime.com/devices/services/689899/d60870b3d706d1c9)
```

Comment:
Displaying the badge without words to describe what it's referring to could be confusing.
I'd like to suggest some text be added to help viewers understand it's meaning.

Something like "Plugin Downloads Server Uptime" -- or whatever's consistent with what's been added to the similar badge that's being added to the cf CLI readme.

Thanks for this PR @a-b !

Reply:
Done, please check and feel free to merge PR if that looks good to you.

Reply:
![image](https://user-images.githubusercontent.com/585662/97611070-efaf4600-19d2-11eb-95fa-08fb34e030bf.png)